examples/run_django_example.sh

```diff
@@ -0,0 +1,11 @@
+cd django-deployment
+make compile-packages
+make install
+brew install redis
+pipenv run python3 manage.py migrate
+pipenv run python manage.py createsuperuser
+FMA_SETTINGS_MODULE=federated_learning_project.fma_settings pipenv run python manage.py runserver &
+redis-server &
+FMA_SETTINGS_MODULE=federated_learning_project.fma_settings pipenv run python manage.py qcluster & fg
```

Comment:
maybe I don't understand this, but look like we are setting `FMA_SETTINGS_MODULE` and then setting it again... 🤔

Reply:
Its two different consoles

Reply:
Not sure if that matters or not inline

Reply:
can do a single live above with `export` technically.